From d2f666f4916d8416f52bc403a5d4046578a1f0a4 Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Tue, 20 Oct 2020 15:45:09 +0700
Subject: [PATCH 328/433] dt-bindings: display: sii902x: Add document for
 sii902x

This patch adds device tree bindings document for sii902x
DVI/HDMI encoder chips

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 .../devicetree/bindings/display/bridge/sii902x.txt | 35 ++++++++++++++++++++++
 1 file changed, 35 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/display/bridge/sii902x.txt

diff --git a/Documentation/devicetree/bindings/display/bridge/sii902x.txt b/Documentation/devicetree/bindings/display/bridge/sii902x.txt
new file mode 100644
index 0000000..c4c1855
--- /dev/null
+++ b/Documentation/devicetree/bindings/display/bridge/sii902x.txt
@@ -0,0 +1,35 @@
+sii902x HDMI bridge bindings
+
+Required properties:
+	- compatible: "sil,sii9022"
+	- reg: i2c address of the bridge
+
+Optional properties:
+	- interrupts: describe the interrupt line used to inform the host
+	  about hotplug events.
+	- reset-gpios: OF device-tree gpio specification for RST_N pin.
+
+Optional subnodes:
+	- video input: this subnode can contain a video input port node
+	  to connect the bridge to a display controller output (See this
+	  documentation [1]).
+
+[1]: Documentation/devicetree/bindings/media/video-interfaces.txt
+
+Example:
+	hdmi-bridge@39 {
+		compatible = "sil,sii9022";
+		reg = <0x39>;
+		reset-gpios = <&pioA 1 0>;
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			port@0 {
+				reg = <0>;
+				bridge_in: endpoint {
+					remote-endpoint = <&dc_out>;
+				};
+			};
+		};
+	};
-- 
2.7.4

